CMakeLists.txt

```diff
@@ -41,3 +43,49 @@ if(CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
     endif()
 endif()
 
+if(LEXY_ENABLE_INSTALL)
```

Comment:
Do we need to have the install available if lexy is used as a subdirectory? I'd prefer if it was moved inside the block like the test/examples/etc. targets.

Reply:
done